linux/systemd/qubes-core.service

```diff
@@ -7,6 +7,8 @@ After=qubes-db-dom0.service libvirtd.service xenconsoled.service
 Type=oneshot
 StandardOutput=syslog
 RemainAfterExit=yes
+# Needed to avoid rebooting before all VMs have shut down.
+TimeoutStopSec=0
```

Comment:
Not sure is this is a good idea. Maybe better simply increase the timeout over default 90s. For example 180s, or 240s.